.github/workflows/preset-image-build.yml

```diff
@@ -131,3 +131,4 @@ jobs:
           hf_username: ${{ secrets.HF_USERNAME }}
           hf_token: ${{ secrets.HF_TOKEN }}
           runs_on: "hostname:model-server"
+          build_target: ${{ matrix.model.build_target }}
```

Comment:
just double checking...for models that don't have build_target specified - will this crash?